build.sbt

```diff
@@ -335,6 +339,37 @@ lazy val blazeCore = libraryProject("blaze-core")
     libraryDependencies ++= Seq(
       blazeHttp
     ),
+    mimaBinaryIssueFilters := {
+      if (tlIsScala3.value)
+        Seq(
+          ProblemFilters
```

Comment:
Could we remove these MiMa filters?

Reply:
Ah yes, all MiMa filters can go 👍

Reply:
I almost committed this too. But thanks, @armanbilge.